README.md

```diff
@@ -63,8 +63,9 @@ programs using Rust. It includes:
 
 `LLVM` is required in your build system to compile BPF bytecode using RedBPF.
 
-- **LLVM 13**  
-  It is needed to compile BPF bytecode.
+- **LLVM 14 or 13**
+  It is needed to compile BPF bytecode. If you're using Rust 1.60 or later, LLVM 14 is required.
+  If you need to use LLVM 13, use the feature `llvm13-0`.
```

Comment:
You seem to have reverted this again?

Reply:
Ah, that's awkward. Made this change on another machine. Sorry about that!

CI configuration needs to be updated too as it seems.

Reply:
> CI configuration needs to be updated too as it seems.

@rsdy I think there are several ways make it work - which one would you recommend to go for?

1. Update the Docker images (needs to be done at some point anyway but now it's a bit more complicated - e.g., `llvm14` isn't packaged for Fedora 35 so you have to use F36 beta).
2. Change build commands in the CI workflows to `cargo build --features=llvm13`.
3. Make `llvm13` the default version/feature for now (I'm favouring this one).

Reply:
@nbaksalyar Sorry for the slow response on this, I've been away from the Internet. Currently bumping images for 1.